### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by adityamedhe**
Have a look

### CODE REVIEW

**Comment by rahulpsd18** [comment]
LGTM 😆

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # ThoughtPool
 
-This repos will contain ideas.
+This repo will contain ideas.
```
Comment (siddharthshishulkar) [inline README.md:3]: 😂 
Well done.

**Review by siddharthshishulkar — APPROVED** [review]
A much-needed change. 🎉

**Comment by adityamedhe** [comment]
Thanks for your reviews. I apologise for not creating an issue before starting on this PR. Will keep in mind the next time @siddharthshishulkar @rahulpsd18 @mahesh1996

**Pull request merged**